### PR TITLE
Document timeout flag for DHCP daemon

### PR DESCRIPTION
**Pull request by tlwr**
What
----

Documents the `timeout` flag from https://github.com/containernetworking/plugins/pull/565

### CODE REVIEW

**Comment by tlwr** [comment]
Rebased after merge conflicts due to https://github.com/containernetworking/cni.dev/pull/79

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-77--cni.netlify.app/plugins/ipam/dhcp/

File: content/plugins/ipam/dhcp.md [hunk 1]
```diff
@@ -24,13 +24,15 @@ $ ./dhcp daemon
 ```
 
 If given `-pidfile <path>` arguments after 'daemon', the dhcp plugin will write
-its PID to the given file.  
+its PID to the given file.
```
Comment (nate-double-u) [inline content/plugins/ipam/dhcp.md:27]:
```suggestion
its PID to the given file.  
```
Adding 2 spaces at the end of the line will tell the markdown parser to add a newline.

File: content/plugins/ipam/dhcp.md [hunk 2]
```diff
@@ -24,13 +24,15 @@ $ ./dhcp daemon
...
 If given `-hostprefix <prefix>` arguments after 'daemon', the dhcp plugin will
 use this prefix for DHCP socket as `<prefix>/run/cni/dhcp.sock`. You can use
 this prefix for references to the host filesystem, e.g. to access netns and the
-unix socket.  
+unix socket.
```
Comment (nate-double-u) [inline content/plugins/ipam/dhcp.md:31]:
```suggestion
unix socket.  
```
Same trick as before, adding 2 spaces will ensure that the next "If given" will appear on the next line.

File: content/plugins/ipam/dhcp.md (hunk 1 above)
Comment (tlwr) [reply content/plugins/ipam/dhcp.md]: Thanks for clarification, resolved in 3bb13d9

**Review by bboreham — APPROVED** [review]
lgtm

**Review by mars1024 — APPROVED** [review]

**Pull request merged**